src/TomlBenchmarks.hs

```diff
@@ -2,5 +2,7 @@ module TomlBenchmarks
        ( someFunc
        ) where
 
+-- TODO: what should I do here? 🤔
```

Comment:
You can delete this file and the `library` stanza from the `.cabal` file. I think it's okay to have only benchmarks in this package.